src/main/java/io/anserini/index/generator/WapoGenerator.java

```diff
@@ -86,6 +86,7 @@ public Document createDocument(WashingtonPostCollection.Document wapoDoc) {
     // This is needed to break score ties by docid.
     doc.add(new SortedDocValuesField(FIELD_ID, new BytesRef(id)));
     doc.add(new LongPoint(WapoField.PUBLISHED_DATE.name, wapoDoc.getPublishDate()));
+    doc.add(new StoredField(WapoField.PUBLISHED_DATE.name, wapoDoc.getPublishDate()));
```

Comment:
I am not sure if `StoreField` is needed. Can you try to remove it to see if it still works?

Reply:
Storefield is needed: 

`An indexed long field for fast range filters. If you also need to store the value, you should add a separate StoredField instance.`

according to http://lucene.apache.org/core/8_2_0/core/org/apache/lucene/document/LongPoint.html